src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenu.cshtml

```diff
@@ -1,9 +1,9 @@
 @inject IAuthorizationService AuthorizationService
 <div class="dropdown">
     <form asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="LogOff" method="post" class="form-inline no-multisubmit">
-        <button type="button" class="ml-3 dropdown-toggle" id="navbarDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" data-toggle-tooltip="tooltip" title="@User.Identity.Name">
+        <button class="unset ml-3 dropdown-toggle" id="navbarDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" data-toggle-tooltip="tooltip" title="@User.Identity.Name">
```

Comment:
While `type` doesn't seem to be required, it's good to include it. Also see: https://dev.to/clairecodes/why-its-important-to-give-your-html-button-a-type-58k9

Reply:
I think you are right. My mistake. I will create another PR.